apisix/debug.lua

```diff
@@ -30,6 +29,7 @@ local setmetatable = setmetatable
 local pcall        = pcall
 local ipairs       = ipairs
 local unpack       = unpack
+local inspect      = require "inspect"
```

Comment:
Better to put all require together, and in the form `require("ngx.process")`

Reply:
`require("apisix.debug")` in `init.lua` --> `require("ngx.process")` in `debug.lua`, will result in error as `process.lua:5: unsupported subsystem: stream`, see: https://github.com/apache/apisix/runs/3578221555

ngx.process API for stream subsystem is not implemented on openresty 1.19.3.2(but is implemented on 1.19.9.1), so I move `require("ngx.process")` to function.

Reply:
Actually I mean to use `require("inspect")`, the `require("ngx.process")` is just an example.